### PR TITLE
Add Circle CI integration

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: circle.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        - sudo add-apt-repository ppa:masterminds/glide -y
+        - sudo apt-get update
+        - sudo apt-get install glide -y
+        - mkdir -p $HOME/go/src/github.com/$CIRCLE_PROJECT_USERNAME
+        - cp -rf $HOME/$CIRCLE_PROJECT_REPONAME $HOME/go/src/github.com/$CIRCLE_PROJECT_USERNAME
+
+test:
+    override:
+        - cd $GS_WD && go test -v $(glide novendor)
+        - cd $GS_WD && go build -a -v -tags netgo -ldflags "-w -X main.gitCommit=$(git rev-parse --short HEAD)"
```
Comment (xh3b4sd) [inline circle.yaml]: Mr. @JosephSalisbury has an opinion here. I think we want to use architect here. See https://github.com/giantswarm/api/blob/master/circle.yml.

**Review by xh3b4sd — CHANGES_REQUESTED** [review]

File: circle.yaml (hunk 1 above)
Comment (rossf7) [reply circle.yaml]: I based this off awstpr. I'll use api instead. Also the filename needs to be circle.yml

**Comment by rossf7** [comment]
Build succeeds now :)

**Comment by JosephSalisbury** [comment]
Hmm, we look for a `main.go` to decide if we should run go build. I guess that doesn't make sense here.

**Comment by asymmetric** [comment]
Closed by #5 I suppose? @JosephSalisbury does it make sense to use `architect` for this repo?

**Comment by JosephSalisbury** [comment]
@asymmetric I'm assuming https://github.com/giantswarm/certificatetpr/pull/5 is a dupe of this PR. Addressed your comments there

**Pull request closed without merging**